jenkins/ci.opensuse.org/openstack-cleanvm.yaml

```diff
@@ -40,8 +41,9 @@
            ( openstack_project=="Newton" && (image=="SLE_12_SP2" ))
         || ( openstack_project=="Ocata" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
         || ( openstack_project=="Pike" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
-        || ( openstack_project=="Queens" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3" || image=="openSUSE-Leap-15.0"))
-        || ( openstack_project=="Master" && (image=="SLE_12_SP3" || image=="SLE_12_SP4" || image=="openSUSE-Leap-42.3" || image=="openSUSE-Leap-15.0"))
+        || ( openstack_project=="Queens" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
+        || ( openstack_project=="Rocky" && (image=="SLE_12_SP4" || image=="openSUSE-Leap-15.0"))
```

Comment:
Why not also test SP3 for now? Is SP4 already far along enough we can focus on it?

Reply:
@nicolasbock we only want one sles and one opensuse image to be used for testing, as it is used to drive the decision to submit from :Staging to NonStaging. by adding more you're not getting consistent stable tree's

Reply:
Thanks!